c++/src/kj/compat/http.c++

```diff
@@ -2637,6 +2713,10 @@ private:
       return bytes[0] & FIN_MASK;
     }
 
+    bool isCompressed() const {
+      return bytes[0] & RSV1_MASK;
+    }
+
     bool hasRsv() const {
```

Comment:
Hmm, it occurs to me that I probably defined this `hasRsv()` so that I could verify that it was always zero but I never actually wrote that check. Maybe we should changes this to `hasRsv2or3()` and actually check it somewhere...

Reply:
We `KJ_REQUIRE` that they're 0 in `receive`, and `KJ_ASSERT` that they're 0 in `send`: https://github.com/capnproto/capnproto/pull/1505/commits/0ca87c76dfb6f70c8b2455998e1302b6f62392ea.